packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.types.ts

```diff
@@ -88,6 +88,11 @@ export interface ISearchBoxProps extends React.InputHTMLAttributes<HTMLInputElem
    */
   clearButtonProps?: IButtonProps;
 
+  /**
+   * Optional name to use for the aria label on clear text button.
```

Comment:
It's not really a name right? It's more a label.

It would be good to document the default value too (via `@default`).